From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 04 Jan 2016 17:19:33 +0000
Subject: latency_hist: Update sched_switch probe

Commit c73464b1c843 ("sched/core: Fix trace_sched_switch()") adds a
'bool preempt' parameter to probe functions, so change
probe_wakeup_latency_hist_stop() accordingly..

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
--- a/kernel/trace/latency_hist.c
+++ b/kernel/trace/latency_hist.c
@@ -116,7 +116,7 @@ static char *wakeup_latency_hist_dir = "
 static char *wakeup_latency_hist_dir_sharedprio = "sharedprio";
 static notrace void probe_wakeup_latency_hist_start(void *v,
 	struct task_struct *p);
-static notrace void probe_wakeup_latency_hist_stop(void *v,
+static notrace void probe_wakeup_latency_hist_stop(void *v, bool preempt,
 	struct task_struct *prev, struct task_struct *next);
 static notrace void probe_sched_migrate_task(void *,
 	struct task_struct *task, int cpu);
@@ -906,7 +906,7 @@ out:
 	raw_spin_unlock_irqrestore(&wakeup_lock, flags);
 }
 
-static notrace void probe_wakeup_latency_hist_stop(void *v,
+static notrace void probe_wakeup_latency_hist_stop(void *v, bool preempt,
 	struct task_struct *prev, struct task_struct *next)
 {
 	unsigned long flags;
